modules/deployment/src/main/java/org/eclipse/xpanse/modules/deployment/deployers/opentofu/opentofulocal/OpenTofuLocalExecutor.java

```diff
@@ -35,6 +38,8 @@ public class OpenTofuLocalExecutor {
     private static final ObjectMapper OBJECT_MAPPER = new ObjectMapper();
     private static final List<String> EXCLUDED_FILE_SUFFIX_LIST =
             Arrays.asList(".tf", ".tfstate", ".binary", ".hcl");
+    private static final int AWAIT_AT_MOST = 60;
```

Comment:
please move this to properties file. so that this can be configurable if necessary.